assets/src/blocks/amp-story-post-date/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { getMetaBlockSettings } from '../../helpers';
+import './edit.css';
+
+export const name = 'amp/amp-story-post-date';
+
+export const settings = {
+	title: __( 'Story Date', 'amp' ),
+	description: __( 'Displays the date the story has been published on', 'amp' ),
+	category: 'common',
+	icon: 'clock',
+	keywords: [
+		__( 'post', 'amp' ),
+		__( 'date', 'amp' ),
```

Comment:
Add `publish` as a keyword.

Reply:
Maybe `publish date`? Otherwise difficult to translate without context

Reply:
Cool. Fixed in 277bb170802910952ee491a6beb11259716bfb63